Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN mkdir -p /home/app/webapp/vendor/bundle && \
 WORKDIR /home/app/webapp
 RUN gem install rubygems-update -v 3.5.6 && \
     gem install bundler:2.5.6 && \
-    /sbin/setuser app bundle install --path vendor/bundle
+    su - app -c "bundle install --path vendor/bundle"
```

Comment:
@ashwinisukale can you add the error you are getting with the old line.

Reply:
[call_build_and_push / build](https://github.com/datacite/poodle/actions/runs/11838356642/job/32987450028#step:7:2033)
buildx failed with: ERROR: failed to solve: process "/bin/sh -c gem install rubygems-update -v 3.5.6 &&     gem install bundler:2.5.6 &&     /sbin/setuser app bundle install --path vendor/bundle" did not complete successfully: exit code: 5

https://github.com/datacite/poodle/actions/runs/11838356642